app/code/Transoft/Blog/Controller/Adminhtml/Index/Upload.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Index;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Catalog\Model\ImageUploader;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Controller\ResultInterface;
+
+/**
+ * Class Upload
+ *
+ * Upload controller class
+ */
+class Upload extends Action implements HttpGetActionInterface
+{
+    /**
+     * Image uploader
+     *
+     * @var ImageUploader
+     */
+    protected $imageUploader;
+
+    /**
+     * Upload constructor.
+     *
+     * @param Context $context
+     * @param ImageUploader $imageUploader
+     */
+    public function __construct(
+        Context $context,
+        ImageUploader $imageUploader
+    ) {
+        parent::__construct($context);
+        $this->imageUploader = $imageUploader;
+    }
+
+    /**
+     * Upload file controller action.
+     *
+     * @return ResultInterface
+     */
+    public function execute()
+    {
+        $imageUploadId = $this->_request->getParam('param_name', 'review_image');
```

Comment:
Weird paramer name *param_name* and default value

Reply:
this looks like default parameter, the same is used anywhere else in Magento vendor modules.